internal/commands/version/version.go

```diff
@@ -0,0 +1,42 @@
+package version
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/rs/zerolog"
+	"github.com/spf13/cobra"
+)
+
+func GetVersion(logger zerolog.Logger) *cobra.Command {
+	cmd := cobra.Command{
+		Use:                   "version",
+		Short:                 "Commands for using parlay with Snyk",
+		Aliases:               []string{"s"},
+		DisableFlagsInUseLine: true,
+		SilenceUsage:          true,
+		Run: func(cmd *cobra.Command, args []string) {
+			tag, err := getCurrentTag()
+			if err != nil {
+				fmt.Println(err)
+			}
+
+			fmt.Println("Current version of the application :", tag)
+		},
+	}
+
+	return &cmd
+}
+
+// git tag --points-at HEAD
+func getCurrentTag() (string, error) {
+	cmd := exec.Command("git", "describe", "--tags", "--abbrev=0")
```

Comment:
This approach will only work when running in the context of the git repository unfortunately,

Reply:
i tried compiling a build and ran this command. This has shown the version @garethr 
![image](https://github.com/snyk/parlay/assets/84926220/3f958d29-acbb-4cf2-8c99-0dbdc14f48db)

Reply:
That's because it's being run in the directory with the Git source. It's picking up the version at runtime, rather than setting the version at build time. If you ran the executable from another directory this wouldn't work unfortunately.